Plugins~/Src/MeshSyncClientBlender/python/unity_mesh_sync_baking.py

```diff
@@ -0,0 +1,1195 @@
+import bpy, os, datetime, time
+from bpy_extras.io_utils import ExportHelper
+from bpy.app.handlers import persistent
+import functools
+
+from .unity_mesh_sync_common import MESHSYNC_PT
+
+# Constants:
+ORIGINAL_MATERIAL = 'ORIGINAL_MATERIAL'
+BAKED_MATERIAL_SHADER = 'BAKED_MATERIAL_SHADER'
+UV_OVERRIDE = 'UV_OVERRIDE'
+
+
+class LogLevel:
+    VERBOSE = 0
+    NORMAL = 1
+    ERROR = 2
+
+
+# For debugging and getting a callstack on error:
+throwExceptions = True
+showLogLevel = LogLevel.VERBOSE
+
+# Commented out ones will be supported in next version:
```

Comment:
Are any of these meant to be commented out?

Reply:
Not anymore. I've removed that comment now.